lib/bindings/HTTPBindings.js

```diff
@@ -244,7 +245,7 @@ function handleIncomingMeasure(req, res, next) {
         }
     }
 
-    utils.retrieveDevice(req.deviceId, req.apiKey, processDeviceMeasure);
+    utils.retrieveDevice(req.deviceId, req.apiKey, transport, processDeviceMeasure);
```

Comment:
What about iota json? Does it have a similar bug?

Reply:
Yes, https://github.com/telefonicaid/iotagent-ul/pull/236#discussion_r158252610 json has the same bug and the some solution could be appplied.

Reply:
So we should create a "paralell" PR to solve this also in the IOTA JSON repo?

Reply:
Yes https://github.com/telefonicaid/iotagent-ul/pull/236#discussion_r158289360 otherwise we need to find a better solution (maybe into iotagent-node-lib)

Reply:
Parallel PR is fine... just not forgetting it once this one gets finished.

Reply:
Do you know how to perform an iota-json autoprovision? I'm not sure if a PR for json is needed.